libethash-cl/CMakeLists.txt

```diff
@@ -17,3 +17,6 @@ TARGET_LINK_LIBRARIES(${EXECUTABLE} ${OpenCL_LIBRARIES} ethash)
 install( TARGETS ${EXECUTABLE} RUNTIME DESTINATION bin ARCHIVE DESTINATION lib LIBRARY DESTINATION lib )
 install( FILES ${HEADERS} DESTINATION include/${EXECUTABLE} )
 
+if (NOT ETHASHCL)
+	install( TARGETS ${LIBRARY} RUNTIME DESTINATION bin ARCHIVE DESTINATION lib LIBRARY DESTINATION lib )
```

Comment:
this will never be executed, see [this](https://github.com/LefterisJP/cpp-ethereum/blob/install_ethash_merge_fix/CMakeLists.txt#L350)